website/docs/r/network_connection_monitor.html.markdown

```diff
@@ -74,41 +157,131 @@ The following arguments are supported:
 
 * `name` - (Required) The name which should be used for this Network Connection Monitor. Changing this forces a new Network Connection Monitor to be created.
 
-* `resource_group_name` - (Required) The name of the Resource Group where the Network Connection Monitor should exist. Changing this forces a new Network Connection Monitor to be created.
-
 * `location` - (Required) The Azure Region where the Network Connection Monitor should exist. Changing this forces a new Network Connection Monitor to be created.
 
-* `destination` - (Required) A `destination` block as defined below.
+* `network_watcher_id` - (Required) The ID of the Network Watcher. Changing this forces a new Network Connection Monitor to be created.
 
-* `network_watcher_name` - (Required) The name of the Network Watcher. Changing this forces a new Network Connection Monitor to be created.
+* `endpoint` - (Required) A `endpoint` block as defined below.
 
-* `source` - (Required) A `source` block as defined below.
+* `test_configuration` - (Required) A `test_configuration` block as defined below.
+
+* `test_group` - (Required) A `test_group` block as defined below.
 
 ---
 
-* `auto_start` - (Optional) Will the connection monitor start automatically once created? Changing this forces a new Network Connection Monitor to be created.
+* `notes` - (Optional) Any notes about the Network Connection Monitor.
 
-* `interval_in_seconds` - (Optional) Monitoring interval in seconds.
+* `output_workspace_resource_ids` - (Optional) A list of the log analytics workspace id that should be used for producing output into a log analytics.
 
 * `tags` - (Optional) A mapping of tags which should be assigned to the Network Connection Monitor.
 
 ---
 
-A `destination` block supports the following:
+A `endpoint` block supports the following:
+
+* `name` - (Required) The name of the connection monitor endpoint.
```

Comment:
Can we capitalize the first letter of the resource type to be consistent (this applies to other parts)?

Reply:
updated